com.chartboost.helium/Android~/helium-android-bridge/src/main/kotlin/com/chartboost/heliumsdk/unity/HeliumUnityAdWrapper.kt

```diff
@@ -0,0 +1,236 @@
+package com.chartboost.heliumsdk.unity
+
+import android.app.Activity
+import android.graphics.Color
+import android.util.DisplayMetrics
+import android.util.Log
+import android.view.Gravity
+import android.view.View
+import android.widget.FrameLayout
+import android.widget.RelativeLayout
+import com.chartboost.heliumsdk.ad.HeliumAd
+import com.chartboost.heliumsdk.ad.HeliumBannerAd
+import com.chartboost.heliumsdk.ad.HeliumBannerAd.HeliumBannerSize
+import com.chartboost.heliumsdk.ad.HeliumFullscreenAd
+import com.chartboost.heliumsdk.ad.HeliumRewardedAd
+import com.unity3d.player.UnityPlayer
+
+class HeliumUnityAdWrapper(private val _ad: HeliumAd?) {
+    // This is the container for the banner.  We need a relative layout to position the banner in one of the 7
+    // possible positions.  HeliumBannerAd is just a FrameLayout.
+    private var mBannerLayout: RelativeLayout? = null
+    private val _activity: Activity = UnityPlayer.currentActivity
+    private val isBanner: Boolean = _ad is HeliumBannerAd
+    private val isFullScreen: Boolean = _ad is HeliumFullscreenAd
+    private var _asBanner: HeliumBannerAd? = null
+    private var _asFullScreen: HeliumFullscreenAd? = null
+    private var startedLoad = false
+
+    private fun ad(): HeliumAd {
+        if (_ad == null) throw RuntimeException("$TAG cannot interact with Helium ad as ad was not created")
+        return _ad
```

Comment:
This can be rewritten as:

```kt
return _ad ?: throw RuntimeException("$TAG cannot interact with Helium ad as ad was not created")
```

Reply:
Ad can never be null so I have removed this.